src/__tests__/FixedSizeList.js

```diff
@@ -421,13 +421,59 @@ describe('FixedSizeList', () => {
       // Scroll down enough to show item 10 at the bottom.
       rendered.getInstance().scrollToItem(10, 'auto');
       // No need to scroll again; item 9 is already visible.
-      // Overscan indices will change though, since direction changes.
+      // Because there's no scrolling, it won't call onItemsRendered.
       rendered.getInstance().scrollToItem(9, 'auto');
       // Scroll up enough to show item 2 at the top.
       rendered.getInstance().scrollToItem(2, 'auto');
       expect(onItemsRendered.mock.calls).toMatchSnapshot();
     });
 
+    it('scroll with align = "auto" should work with partially-visible items', () => {
+      const rendered = ReactTestRenderer.create(
+        // Create list where items don't fit exactly into container.
+        // The container has space for 3 1/3 items.
+        <FixedSizeList {...defaultProps} itemSize={30} />
+      );
+      // Scroll down enough to show item 10 at the bottom.
+      // Should show 4 items: 3 full and one partial at the end
+      rendered.getInstance().scrollToItem(10, 'auto');
+      // No need to scroll again; item 9 is already visible.
+      // Because there's no scrolling, it won't call onItemsRendered.
+      rendered.getInstance().scrollToItem(9, 'auto');
+      // Scroll to near the end. #96 will be shown as partial.
+      rendered.getInstance().scrollToItem(99, 'auto');
+      // Scroll back to show #96 fully. This will cause #99 to be shown as a
+      // partial. Because #96 was already shown previously as a partial, all
+      // props of the onItemsRendered will be the same. This means that even
+      // though a scroll happened in the DOM, the onItemsRendered event is
+      // not triggered. IMHO this is probably a bug, but it's one we can fix
```

Comment:
This would be working as per the spec. Why do you consider this a bug? `onItemsRendered` should only be called when the rendered indicates change.

Reply:
Yep, you're right. At the time I wrote the comment above I thought that `onItemsRendered` was expected to be called when an item changed from being partially to fully visible. Sorry I forgot to update that comment!